mautic/mautic
# Fix/change hubspot api auth 5.x

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ x ] 
| Issue(s) addressed                     | Fixes #11767  



#### Description:
Cherry picked commits to address Hubspot changes, which are available only in 4.4.
We need to change authentication method for Hubspot plugin. New installation cannot use our plugin anymore, and existing in integration will shut down in November, 22'.


#### Steps to test this PR:
Same as in #11470

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a private app in hubspot with following rights
3. Setup plugin integration
4. See everything works
